pyt/vulnerabilities.py

```diff
@@ -466,7 +469,8 @@ def find_vulnerabilities_in_cfg(
     lattice,
     ui_mode,
     blackbox_mapping,
-    vulnerabilities_list
+    vulnerabilities_list,
+    nosec_lines = set()
```

Comment:
So this shouldn't be necessary, (to make it default to empty set), we can just leave it as `nosec_lines`. This is b/c we only call `find_vulnerabilities` from tests, not this function.